docs/architecture/blazor-for-web-forms-developers/app-startup.md

```diff
@@ -1,15 +1,108 @@
 ---
 title: App startup
 description: Learn how to define the startup logic for your app.
-author: danroth27
-ms.author: daroth
-ms.date: 09/11/2019
+author: csharpfritz
+ms.author: jefritz
+ms.date: 02/25/2020
 ---
 # App startup
 
 [!INCLUDE [book-preview](../../../includes/book-preview.md)]
 
-*This content is coming soon.*
+Applications that are written for ASP.NET typically have a `global.asax.cs` file that defines the `Application_Start` event and controls services are configured and made available for both HTML rendering and .NET processing.  This chapter looks at how things are slightly different with ASP.NET Core and Blazor Server.
+
+## Application_Start and Web Forms
+
+The default web forms `Application_Start` method has grown in purpose over the last 10 years of development to handle many configuration task.  A fresh web forms project with the default template in Visual Studio 2019 now contains the following configuration logic:
+
+- `RouteConfig` - Application URL routing
+- `BundleConfig` - CSS and JavaScript bundling and minification
+
+Each of these individual files reside in the `App_Start` folder and run only once at the start of our application.  `RouteConfig` in the default project template adds the `FriendlyUrlSettings` for web forms to allow application URLs to omit the `.ASPX` file extension.  The default template also contains a directive that provides permanent HTTP redirect status codes (HTTP 301) for the `.ASPX` pages to the friendly URL with the file name that omits the extension.
+
+With ASP.NET Core and Blazor, these methods are either simplified and consolidated into the `Startup` class or they are eliminated in favor of common web technologies.
+
+## Blazor Server Startup Structure
```

Comment:
Now that Blazor WebAssembly has shipped, I think we need a Blazor WebAssembly section as well. Blazor WebAssembly apps define startup logic in Program.Main and don't use the same `Startup` pattern used by ASP.NET Core. Technically both Blazor Server and Blazor WebAssembly define a `Program.Main` that sets up the host for the app. Blazor Server apps use the normal ASP.NET Core host, while Blazor WebAssembly apps have a WebAssembly specific host.

Reply:
This feels like scope creep at this point, and I suggest we recommend adding a chapter or two after the first version ships

Reply:
We already talk about Blazor WebAssembly in the earlier chapters in the book, but we can unblock this PR and add the content later. Filed https://github.com/dotnet/docs/issues/19842 to track.